troposphere/static/js/components/admin/ResourceAdmin.react.js

```diff
@@ -13,7 +14,12 @@ define(function (require) {
 
     getInitialState: function () {
       return {
-        response: ""
+        AUSearch: "",
+        delta: 525600,
```

Comment:
The 525600 is there as a default since it is the value that most current allocations have for delta. The actual value doesn't matter as long as it's not -1, but since allocation objects already exist with 525600 it's easier to just use them instead of making new ones with something like delta 0.